geps/gep-713.md

```diff
@@ -560,50 +560,536 @@ ties:
+months ago. And there are lots of different kinds of policy and lots of
+individual policies, hang on a minute..._
+
+**Charlie**: _Looks like about 47 for your chunk of the world, a couple hundred
+system-wide._
+
+**Ana**: 😱 _Can you tell me what they’re doing for each of our services? I
+can’t even_ look _at these things._ 😕
+
+**Charlie**: _That's gonna take awhile. Our tooling to show us which policies
+bind to a given workload doesn't go the other direction._
+
+**Ana**: _...wait. You have to_ build tools _to know if retries are turned on??_
+
+Pause.
+
+**Charlie**: _Policy attachment is more complex than we’d like, yeah._ 😐
+_Look, how ‘bout roll back your `baker` change for now? We can get together in
+the morning and start sorting this out._
+
+Ana shakes her head and rolls back her edits to the `baker` Deployment, then
+sits looking out over the lake as the deployment progresses.
+
+**Ana**: _Done. Are things happier now?_
+
+**Charlie**: _Looks like, thanks. Reckon you can get back to your sailboard._ 🙂
+
+Ana sighs.
+
+**Ana**: _Wish I could. Wind’s died down, though, and it'll be dark soon.
+Just gonna head home._
+
+**Charlie**: _Ouch. Sorry to hear that._ 😐
+
+One more look out at the lake.
+
+**Ana**: _Thanks for the help. Wish we’d found better answers._ 😢
+
+### The Problem, restated
+What this parable makes clear is that, in the absence of information about what
+Policy is affecting an object, it’s very easy to make poor decisions.
+
+It’s critical that this proposal solve the problem of showing up to three things,
+listed in increasing order of desirability:
+- _That_ some Policy is affecting a particular object
+- _Which_ Policy is (or Policies are) affecting a particular object
+- _What_ settings in the Policy are affecting the object.
+
+In the parable, if Ana and Charlie had known that there were Policies affecting
+the relevant object, then they could have gone looking for the relevant Policies
+and things would have played out differently. If they knew which Policies, they
+would need to look less hard, and if they knew what the settings being applied
+were, then the parable would have been able to be very short indeed.
+
+(There’s also another use case to consider, in that Charlie should have been able
+to see that the Policy on the namespace was in use in many places before deleting
+it.)
+
+To put this another way, Policy Attachment is effectively adding a fourth Persona,
+the Policy Admin, to Gateway API’s persona list, and without a solution to the
+discoverability problem, their actions are largely invisible to the Application
+Developer. Not only that, but their concerns cut across the previously established
+levels.
+
+![Gateway API diagram with Policy Admin](images/713-the-diagram-with-policy-admin.png)
+
+
+From the Policy Admin’s point of view, they need to know across their whole remit
+(which conceiably could be the whole cluster):
+- _What_ Policy has been created
```

Comment:
Needed for list formatting in our markdown parser
```suggestion
(which conceiably could be the whole cluster):

- _What_ Policy has been created
```

Reply:
Fixed, I fixed the typo as well. ("conceiably")